lib/linux_admin/time_date.rb

```diff
@@ -6,10 +6,12 @@ class TimeDate
     TimeCommandError = Class.new(StandardError)
 
     def self.system_timezone_detailed
-      result = run(cmd(COMMAND), :params => ["status"])
+      result = run!(cmd(COMMAND), :params => ["status"])
       result.output.split("\n").each do |l|
         return l.split(':')[1].strip if l =~ /Time.*zone/
       end
+    rescue AwesomeSpawn::CommandResultError, AwesomeSpawn::NoSuchFileError
+      Time.now.zone
```

Comment:
This isn't the same result as what we would get from `timedatectl`.
I'm not sure if that matters, but I don't know of a good way to convert between the `America/New_York` style and `EST`